pages/about-us/donate/inc/_faq.en.text

```diff
@@ -1,6 +1,6 @@
-h2. How do I do a wire/bank transfer?
+h2. Why don't you accept wire transfers in other currencies?
 
-If you want to make an ACH wire transfer, please contact us at treasurer@riseup.net for information on our ABA/SWIFT/IBAN number. *Please note:* We have to pay $13 for every incoming wire transfer and it is often very difficult for people to successfully transfer money to us this way. We would therefore encourage you to use another method.
+We are charged a large fee for converting other currencies. Additionally, sending directly to our US credit union often is difficult and does not work, and when it does work, we get charged a $13 fee for each transfer, so it is not economical.
 
 h2. How do I donate from outside the U.S?
```

Comment:
This section "How do I donate from outside the U.S." doesn't quite make sense now because we have a way for EUR donations to come in from outside the US. We either should make the first item in the list say, "If you want to send us EUR, then please see the "Wire/Bank transfers (EUR only)" section" with a link to it.

Or we could change the heading to instead say "How do I donate in other currencies than USD, or EUR?"